debian/rules

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --builddirectory=_build
+
+override_dh_auto_configure:
+	mkdir _build && cd _build && ../configure --prefix=/usr
```

Comment:
Debian supports multiarch, so it should probably specify the `--libdir` accordingly. I'm not fluent in Debian packaging, though, so you may need to ask Dan.